spec.md

```diff
@@ -155,6 +155,8 @@ These attributes, while descriptive of the event, are designed such that they
 can be serialized independent of the event data. This allows for them to be
 inspected at the destination without having to deserialize the event data.
 
+#### Type System
```

Comment:
@cneijenhuis let me know when you've changed this to `###` and then I'll merge

Reply:
@duglin Done :)